xbmc/addons/AddonDll.cpp

```diff
@@ -425,5 +523,233 @@ bool CAddonDll::CheckAPIVersion(int type)
   return true;
 }
 
+/*!
+ * @brief Addon to Kodi basic callbacks below
+ *
+ * The amount of functions here are hold so minimal as possible. Only parts
+ * where needed on nearly every add-on (e.g. addon_log_msg) are to add here.
+ *
+ * More specific parts like e.g. to open files should be added to a separate
+ * part.
+ */
+//@{
+ 
+bool CAddonDll::InitInterface(KODI_HANDLE firstKodiInstance)
+{
+  memset(&m_interface, 0, sizeof(m_interface));
```

Comment:
Is removed on this file and also removed them on Screensaver class who was brought before.